src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -47,20 +46,20 @@ public void outputLine(String line) {
     }
   };
 
-  @Component
+  @Parameter(defaultValue = "${pluginDescriptor}", readonly = true)
```

Comment:
It's the same thing. `@Component ` is now deprecated. Hence the change. `readonly=true` makes it invisible to users of the plugin.